### ISSUE TITLE
Question: rudy-* dependencies

### ISSUE THREAD

**Issue by ymaz** [faceyspacey/redux-first-router#341]
Hi, I've spotted that `redux-first-router` has two dependencies:

- rudy-history;
- rudy-match-path;

As I understand, that it's two forks of original packages `history` and `match-patch`.

So, my question: is there a reason that you are using forks of those dependencies instead of original dependencies? It's kind of scary to use both main dependencies as your forks.

Thanks,

**Comment by ScriptedAlchemy** [comment]
Yeah, we actually have overhauled these in later versions of Rudy, Which is its own thing. We started overhauling them in RFR 2. https://github.com/respond-framework/rudy

There are quite a few dependencies that are pretty much under either myself of faceyspacey's direct control and nobody else. 

There's a substantial amount of publically traded companies dependent on RFR, no complaints yet :)

**Comment by ScriptedAlchemy** [comment]
As for what we are doing in the forks, to be honest this was a little before my time. I only started contributing to Rudy after it had already undergone substantial structural changes. I can diff the forks and see why we did fork the repos back then. You can replace Rudy history with the usual history package though. People have done it without much hassle. I know why Rudy has its own history management, but off the top of my head I’m not sure what RFR does with it. I spend lots of time on Rudy, so the lines blur between the two and need to sit down at my machine to give you a factual response

**Comment by ymaz** [comment]
hey @ScriptedAlchemy thanks for looking into that.

>You can replace Rudy history with the usual history package though. People have done it without much hassle. 

But, in that case I will have my own fork of `redux-first-router` :)

>I can diff the forks and see why we did fork the repos back then.

So, it would be nice if we are able to use non-fork dependency. Is that possible to investigate?

Thanks,

**Comment by hedgepigdaniel** [comment]
I think it would be a good idea to work out what the differences are, try to get any necessary features merged into the upstreams, and switch back to them.

That's assuming the difference is reasonably small and that is a realistic thing to do.
